homeassistant/components/netatmo/manifest.json

```diff
@@ -11,5 +11,11 @@
     "models": ["Healty Home Coach", "Netatmo Relay", "Presence", "Welcome"]
   },
   "iot_class": "cloud_polling",
-  "loggers": ["pyatmo"]
+  "loggers": ["pyatmo"],
+  "supported_brands": {
+    "legrand": "Legrand",
```

Comment:
We also support Legrand Home+ Control; wouldn't this be highly confusing to list it next to this?

![image](https://user-images.githubusercontent.com/195327/193993346-772d1ad3-340d-4fc8-a96c-e87b6c7482cb.png)

Reply:
I agree, but Legrand Home + Control (API) will be deprecated. See https://developer.legrand.com/forums/topic/deprecation-of-home-control-api-on-works-with-legrand/

Reply:
What do you suggest?

Reply:
I really don't have a suggestion, I just see a collision/unclarity for the end-user. I'm not familiar with the products(lines), APIs, or things Netatmo will do with (or with which) Legrand devices. So, besides signaling a possible issue for the end-user, I'm not able to provide a suggestion.

Sorry.

../Frenck

Reply:
As discussed on Discord, we'll go ahead with this for now.